src/smart-components/access-management/UsersTable.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useCallback } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useDataViewSelection, useDataViewPagination } from '@patternfly/react-data-view/dist/dynamic/Hooks';
+import { BulkSelect, BulkSelectValue } from '@patternfly/react-component-groups/dist/dynamic/BulkSelect';
+import { DataView } from '@patternfly/react-data-view/dist/dynamic/DataView';
+import { DataViewToolbar } from '@patternfly/react-data-view/dist/dynamic/DataViewToolbar';
+import { DataViewTable } from '@patternfly/react-data-view/dist/dynamic/DataViewTable';
+import { Pagination } from '@patternfly/react-core';
+import { ActionsColumn } from '@patternfly/react-table';
+import { fetchUsers } from '../../redux/actions/user-actions';
+import { mappedProps } from '../../helpers/shared/helpers';
+import { RBACStore } from '../../redux/store';
+
+interface User {
```

Comment:
It's just nitpicking, but not sure if here is the best place to define this interface - it may be reused in multiple places. How about moving it to the user-reducer or similar place?

Reply:
Updated to use the existing interface in `/user/user-table-helpers.tsx`